.github/workflows/docs_test.yml

```diff
@@ -0,0 +1,57 @@
+name: docs_test
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+
+concurrency:
+  group: docs_test-${{ github.event.pull_request.number || github.ref }}
+  cancel-in-progress: true
+
+env:
+  GO_VERSION: 1.14.2
+
+jobs:
+  docs_test:
+    name: docs_test
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out the codebase.
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: ${{ env.GO_VERSION }}
+
+      - name: Caching dependency
+        uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cache/go-build
+            ~/go/pkg/mod
+            ~/go/bin
+          key: ${{ runner.os }}-v1-go-mod-${{ hashFiles('**/go.mod') }}-${{ hashFiles('**/go.sum') }}
+
+      - name: go mod download
+        run: |
+          [[ -d ~/go/pkg/mod ]] && exit 0
+          # retry up to 3 times in case of network issues
+          for i in $(seq 1 3); do
+              go mod download && exit 0
+              sleep 10
+          done
+          exit 1
+
+      - name: Install gomplate
+        run: .github/scripts/install-gomplate.sh
```

Comment:
Let's just move the installation commands to this step and delete the `install-gomplate.sh` script.

Reply:
As per suggestion, I have moved installation commands to the workflow step itself. Because `install-gomplate.sh` is not used by any other workflows.